src/components/FlawForm.vue

```diff
@@ -222,6 +223,9 @@ const createdDate = computed(() => {
               Open in Bugzilla <i class="bi-box-arrow-up-right ms-2" />
             </a>
           </div>
+          <div>
+            <FlawHistory :flaw="flaw" @expandFocusedComponent="expandFocusedComponent" />
+          </div>
```

Comment:
Do we need this div? the `FlawHistory` component already has a root element

Reply:
Removed from both, Alerts and History